packages/rum/src/userActionCollection.ts

```diff
@@ -9,20 +9,22 @@ export enum UserActionType {
   CUSTOM = 'custom',
 }
 
+type AutoUserActionType = UserActionType.CLICK
```

Comment:
LGTM for me ! Just one thing : Considering Benoit's work to rename `UserAction` to `Action` in web-ui, shouldn't we do the same here and rename `AutoUserActionType` => `AutoActionType`... ?

Reply:
Let's wait for v2 migration inside the SDK to do this cleaning